### PR TITLE
fix(notebook): put id column as str and explain density representation

### PR DESCRIPTION
**Pull request by albanfelix**
Related to #2236

Modified df fixtures now have ID_VARIABLE column as type string.

Tuto 9 modifications are just lint-fix

### CODE REVIEW

File: notebooks/Tutorial9-Multitable.py [hunk 1]
```diff
@@ -6,10 +6,6 @@
 #       format_name: light
```
Comment (albanfelix) [inline notebooks/Tutorial9-Multitable.py:6]: Tuto 9 modifications are just `lint-fix`

File: notebooks/Tutorial10_Geolocation_privacy_assessment.py [hunk 2]
```diff
@@ -109,9 +102,27 @@
...
+# - Histogram: In this method, the area is divided into a grid of cells, and the number of data points falling into each cell is counted. The count is then used to assign a color to each cell, creating a heatmap. Cells with a higher count will have a darker color, indicating a higher density of data points in that region.
+# - Kernel Density Estimation: This method uses a statistical technique to estimate the underlying probability density function of the data. It involves placing a kernel (a smooth, continuous function) on each data point and summing up the contributions of all kernels to create a smooth density estimate. The density estimate is then used to generate a heatmap, where regions with higher density have a darker color.
+#
+# Both methods provide a visual representation of the density of data points, but they differ in terms of how they estimate the density. Histograms are simpler and provide a more discrete representation, while kernel density estimation produces a smoother and continuous density estimate.
+#
+# Here is the density representation of all Porto taxi' traces:
+
+# ![Density representation of Porto' traces£](img/porto_density.png)
```
Comment (olivierabz) [inline notebooks/Tutorial10_Geolocation_privacy_assessment.py:117]: I actually think that we should not include png here but something generated from the data. This has more its place in the doc (https://docs.octopize.io/docs/understanding/geolocation_privacy_assessment)

**Review by olivierabz — CHANGES_REQUESTED** [review]
The explanations and figures are really clear but I believe they should go in the doc (https://docs.octopize.io/docs/understanding/geolocation_privacy_assessment) rather than the notebook.

File: notebooks/Tutorial10_Geolocation_privacy_assessment.py [hunk 3]
```diff
@@ -87,7 +80,7 @@
 plt.show
 
 # +
-selected_id = 1
+selected_id = "id1"
```
Comment (olivierabz) [inline notebooks/Tutorial10_Geolocation_privacy_assessment.py:83]: Also I disagree with this. We should not have to change the data file. We need to go and fix the ID handling, for sure we'll get numeric IDs in practice.

**Pull request closed without merging**